privacyidea/translations/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,2717 @@
+"zu verarbeiten."
+
+#: lib/policy.py:1917
+msgid ""
+"If the administrator sets the OTP PIN during enrollment or later, the user will "
+"have to change the PIN during first use."
+msgstr ""
+"Wenn der Administrator die OTP PIN setzt (während des Ausrollens oder später) "
+"muss der Benutzer die OTP PIN zurücksetzen."
+
+#: lib/policy.py:1924
+msgid ""
+"The user needs to change his PIN on a regular basis. To change the PIN every 180 "
+"days, enter '180d'."
+msgstr ""
+"Der Benutzer muss die OTP PIN regelmäßig ändern. Um die PIN all 180 Tage zu "
+"ändern, geben Sie '180d' ein."
+
+#: lib/policy.py:1931
+msgid ""
+"The OTP PIN can be hashed or encrypted. Hashing the PIN is the default behaviour."
+msgstr ""
+"Die OTP PIN wird standardmäßig gehasht abgespeichert. Wenn sie verschlüsselt "
+"abgespeichert werden soll, wählen Sie diese Option."
+
+#: lib/policy.py:1936
+msgid ""
+"Set label for a new enrolled Google Authenticator. Possible tags are <u> (user), "
+"<r> (realm), <s> (serial)."
+msgstr ""
+"Setzt die Beschriftung für einen neu ausgerollten Google Authenticator oder "
+"vergleichbare App. Mögliche Tags sind <u> (Benutzer), <r> (Realm) oder <s> "
+"(Seriennummer)."
+
+#: lib/policy.py:1942
+msgid "This is the issuer label for new enrolled Google Authenticators."
+msgstr ""
+"Die Bezeichnung des Ausstellers für neu ausgerollte Google Authenticator oder "
+"vergleichbare Apps."
+
+#: lib/policy.py:1948
+msgid "This is the URL to the token image for smartphone apps like FreeOTP."
+msgstr "URL zu einem Bild, das zum Token in einer App wie FreeOTP angezeigt wird."
+
+#: lib/policy.py:1955
+msgid ""
+"Users can assign a token just by using the unassigned token to authenticate."
+msgstr ""
+"Der Benutzer kann sich automatisch einen Token zuweisen, indem er sich mit einem "
+"noch nicht zugewiesenen Token anmeldet."
+
+#: lib/policy.py:1961
+msgid "The length of the password in case of temporary token (lost token)."
+msgstr ""
+"Die Länge des Passworts für einen temporären Token (Workflow \"Verlorener Token"
+"\")"
+
+#: lib/policy.py:1965
+msgid ""
+"The contents of the temporary password, described by the characters C, c, n, s, "
+"8."
+msgstr ""
+"Der Inhalt des temporären Passworts. 'C': Große Buchstaben, 'c': kleine "
+"Buchstaben, 'n': Ziffern, 's': Sonderzeichen, '8' Base58 Zeichensatz."
+
+#: lib/policy.py:1970
+msgid "The length of the validity for the temporary token (in days)."
+msgstr "Die Gültigkeitsdauer des temporären Passworts in Tagen."
+
+#: lib/policy.py:1975
+msgid "Set the length of registration codes."
+msgstr "Setzt die Länge des Registrierungscodes."
+
+#: lib/policy.py:1979
+msgid ""
+"Specify the required contents of the registration code. (c)haracters, (n)umeric, "
+"(s)pecial. Use modifiers +/- or a list of allowed characters [1234567890]"
+msgstr ""
+"Definiert den erforderlichen Inhalt des Registrierungscodes. Zeichen (c), "
+"Nummern (n) und Spezialzeichen (s). Die Modifikatoren +/- können verwendet "
```

Comment:
shouldn't this be "Sonderzeichen"? I think this is the common term.

Reply:
You are totally correct (aka right) ;-)

Reply:
Done in ee9d9bff